docs/wiki/Getting started - Get module cross-references.md

```diff
@@ -0,0 +1,40 @@
+The `'Get-LinkedLocalModuleList'` function provides you with the capability to check for any local module references in a given path. This can be useful to determine which modules folder you'd need if you don't want to keep the entire library.
+
+---
+
+### _Navigation_
+
+- [Location](#location)
+- [How it works](#what-it-does)
+- [How to use it](#how-to-use-it)
+- [Related function: _Get-LinkedModuleList_](#related-function-get-linkedmodulelist)
+
+---
+# Location
+
+You can find the script under `'utilities/tools/Get-LinkedLocalModuleList.ps1'`
+
+# How it works
+
+When invoked, the script
+
+1. The function leverages the utility [Get-LinkedModuleList](#related-function-get-linkedmodulelist) to fetch all references implemented in the modules in a given path
+1. The function filters these references down to only local references (i.e. cross-module references) and formats them to show a consistent '\<ProviderNameSpace\>/\<ResourceType\>' format.
+1. Finally, it prints the references to the invoking terminal, group by ResourceType.
+
+# How to use it
+
+> **Note:** The script must be loaded before the function can be invoked
+
+For details on how to use the function please refer to the script's local documentation.
+
+# Related function: _Get-LinkedModuleList_
+
+The function `'Get-LinkedModuleList'` (also in path `'utilities/tools'`) is leveraged by the `'Get-LinkedLocalModuleList'` function, but can also be invoked on its own. You can use it to get an overview of all references implemented in any module in a given path. This includes:
+- Resource deployments
+- Cross-Module references
+- Remove-Module references (e.g., Bicep Registry)
+
+> **Note:** The script must be loaded before the function can be invoked
+
+For details on how to use the function please refer to the script's local documentation.
```

Comment:
Can we add a small output example?

Reply:
We have an example in the script. Are you sure we want to duplicate the content? Having the ReadMe already feels like a stretch to maintain.

Reply:
I think the 'cross references' mechanism we are introducing deviates a bit from the 'atomic unit' that we originally had as now we have dependencies outside of the main module. The guidance you've written is pretty good, but the diagram you have in the description in this PR is also useful. I see this page is going to be frequently visited as a lot of core modules support Private endpoints. So a visual / output example is good to have.

Reply:
I agree with @ahmadabdalla. Not about the function, that is how we document other utilities as well. But regarding the overall approach I think we should add a dedicated section for cross references in the module design page.

Reply:
I agree, but the diagram was created manually and would quickly become stale. The output generated by the function isn't the mermaid code. Anyways, if you feel it's worthwhile I'll add a sample.

Reply:
not a sample, we need to document the design, how cross referencing modules works. I wouldn't list the modules using pe in that way, that for sure will get stale.

Reply:
added

Reply:
@MrMCake I'm good with the addition you've made. Thanks. Leaving it to @eriqua for further feedback.